### PR TITLE
 feat(admin-ui): update identity page

### PR DESCRIPTION
**Pull request by frdomovic**
# feat(admin-ui): update identity page

## Summary
New design + api handler for did calls

Fixes # C20-63

## Summary by CodeRabbit

- **New Features**
  - Introduced `IdentityRowItem` component for displaying identity information in a structured row format.
  - Added new API method `getDidList` to fetch a list of root keys.

- **Updates**
  - Replaced `IdentityContent` with `IdentityTable` for improved data handling and display.
  - Revamped the `IdentityTable` component with updated structure, styling, and logic.
  - Updated terminology from `RootKey` to `RootKeyObject`.

- **Localization**
  - Updated global English translations with new keys for the identity page.

- **Refactor**
  - Removed unused components and imports for cleaner codebase.
  - Adjusted import orders and refactored data mapping functions for better organization.

### CODE REVIEW

File: node-ui/src/api/dataSource/NodeDataSource.ts [hunk 1]
```diff
@@ -110,4 +114,18 @@ export class NodeDataSource {
...
+      const response = await this.client.get<any>("/admin-api/did");
+      if (response?.data?.root_keys) {
+        return response.data.root_keys;
+      } else {
+        return [];
+      }
+    } catch (error) {
+      console.error("Error fetching installed applications:", error);
+      return [];
+    }
```
[the remark this answers is not in the document]
Comment (frdomovic) [reply node-ui/src/api/dataSource/NodeDataSource.ts]: updated

File: node-ui/src/components/identity/IdentityRowItem.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from "styled-components";
+import MenuIconDropdown from "../common/MenuIconDropdown";
+import { RootKeyObject } from "../../utils/rootkey";
+
+interface RowItemComponentProps {
+  $borders: boolean;
+}
```
[the remark this answers is not in the document]
Comment (frdomovic) [reply node-ui/src/components/identity/IdentityRowItem.tsx]: updated

File: node-ui/src/api/dataSource/NodeDataSource.ts [hunk 3]
```diff
@@ -20,6 +20,21 @@ export interface ContextsList<T> {
   invited: T[];
 }
 
+export interface RootKey {
+  signing_key: string;
+}
+
+interface ClientKey {
+  signing_key: string;
```
Comment (MatejVukosav) [inline node-ui/src/api/dataSource/NodeDataSource.ts:32]: camelCase

Comment (frdomovic) [reply node-ui/src/api/dataSource/NodeDataSource.ts:32]: Screenshot 2024-05-17 at 12 21 03
Screenshot 2024-05-17 at 12 21 48

Not sure what should be changed here as this is returned from the api and needs to have a type here.
Screenshot 2024-05-17 at 12 23 04

I updated the "signing_key" to camel case, but "client_keys" and "context" are not in used in identity page

**Pull request merged**